app.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const app = express();
+const usersList = require("./userService");
+
+app.use(express.static('public'));
+
+app.get("/", function(req, res) {
+    res.render("index.pug");
+});
+
+// Load page
+app.get("/usersCenter", function(req, res) {
+    res.render("usersCenter.pug");
+});
+
+// Get Users
+app.get("/users", function(req, res) {
+    usersList.getUsers(req.query, res);
+});
+
+// Create a User
+app.post("/users", function(req, res) {
+    res.sendStatus(200);
```

Comment:
status 200 is default. But for endpoints that creates a resource it must be 201
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
Fixed.